### ISSUE TITLE
Improve the Documentation 

### ISSUE THREAD

**Issue by manalErpnext** [frappe/erpnext#42810]
I hope to be the documentation less boring than this
could make an app with a competitive environment between the members, and the discussion of questions within the application

**Comment by doancan** [comment]
I completely agree with you, but I think it would make more sense to make this suggestion on the Frappe Wiki repository page.
